numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2887,8 +2887,16 @@ def add_newdoc(place, name, doc):
 
     Computes the remainder complementary to the `floor_divide` function.  It is
     equivalent to the Python modulus operator``x1 % x2`` and has the same sign
-    as the divisor `x2`. It should not be confused with the Matlab(TM) ``rem``
-    function.
+    as the divisor `x2`.
+
+    .. warning::
+
+        This should not be confused with:
+
+        * Python 3.7's `math.remainder` and C's ``remainder``, which
+          computes the IEEE remainder, which are the complement to
+          ``round(x1 / x2)``.
+        * Matlab's ``rem``, which is the complement to ``int(x1 / x2)``.
```

Comment:
Does this need the (TM) appendix? Just curious because the original version had one.

Reply:
That was accidental - but doing a search seems to reveal that we don't use a TM sign anywhere else.

Reply:
There are other places, but all in the ufunc documentation. It looks like we don't need it for the function documentation, https://www.mathworks.com/brandguide/editorial/products.html, but we might want to put a copyright and trademark notice somewhere in the documentation, I don't know where.

Reply:
Note that the mathworks style guide wants it in caps and without the possessive suffix.

Reply:
Ok, I've switched to all caps